sparksteps/steps.py

```diff
@@ -155,13 +165,19 @@ def upload_steps(s3_resource, bucket, bucket_path, src_path):
     """Upload files to S3 and get steps."""
     steps = []
     basename = get_basename(src_path)
-    if os.path.isdir(src_path):  # zip directory
-        copy_step = CopyStep(bucket, bucket_path, basename + '.zip')
-        zip_to_s3(s3_resource, src_path, bucket, key=copy_step.key)
+    default_dest_path = os.path.join(bucket_path, 'sources')
```

Comment:
@amajedi I think we should clean up and rename this function a bit. Thinking more about it, we're really doing downloads to `REMOTE_DIR` with uploads being an intermediate.
I'd rename `upload_steps` to `download_steps` and add some comments to what's going on.

Reply:
Seems a bit out of scope of this PR (since function already behaved this way), but I went ahead and renamed the function to `get_download_steps` and added a few comments anyway.